runtime/Cpp/runtime/src/Recognizer.h

```diff
@@ -11,7 +11,7 @@ namespace antlr4 {
 
   class ANTLR4CPP_PUBLIC Recognizer {
   public:
-    static const size_t EOF = std::numeric_limits<size_t>::max();
+    static const size_t EOF = (size_t)-1; // std::numeric_limits<size_t>::max(); doesn't work in VS 2013.
```

Comment:
Would it be possible to use `static_cast<size_t>(-1)` as an alternative? (here are elsewhere where there are old-style casts in this patch.)

Also: there is the `INVALID_INDEX` #define at the end of antlr-common.h, which uses the same construct. Does VS2013 choke on use of that as well?